packages/terra-core-docs/src/terra-dev-site/doc/button-group/example/ButtonGroupIconA11y.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import IconPrinter from 'terra-icon/lib/icon/IconPrinter';
+import IconSave from 'terra-icon/lib/icon/IconSave';
+import ButtonGroup from 'terra-button-group';
+
+const iconPrint = <IconPrinter a11yLabel="Print" />;
+const iconSave = <IconSave a11yLabel="Save" />;
+
+const ButtonGroupA11yIcon = () => (
+  <ButtonGroup>
+    <ButtonGroup.Button text="All Documents" icon={iconPrint} key="print" />
+    <ButtonGroup.Button text="All Documents" icon={iconSave} key="save" />
```

Comment:
```suggestion
    <ButtonGroup.Button text="Print Documents" icon={iconPrint} key="print" />
    <ButtonGroup.Button text="Save Documents" icon={iconSave} key="save" />
```

Reply:
I thought it will read the phrase together along with the icon A11yLabel. Looks like thats not the case . Changed here [49a1ff1](https://github.com/cerner/terra-core/pull/3679/commits/49a1ff169dfc61ea67a1942fa95ace533ef904c9)